learned.lic

```diff
@@ -4,13 +4,15 @@
 
 custom_require.call(%w(drinfomon))
 
+learning_time = ((Time.now - DRSkill.start_time) / 60.0 / 60).round(1)
+
 DRSkill.list.select { |item| item.current - item.baseline > 0 }.sort_by { |item| item.current - item.baseline }.reverse.each_slice(4) do |skills|
   output = ''
-  skills.each { |skill| output += skill.name.center(18) + DRSkill.gained_exp(skill.name).to_s.center(6) }
+  skills.each { |skill| output += skill.name.center(18) + DRSkill.gained_exp(skill.name).to_s.center(6) + "(#{(DRSkill.gained_exp(skill.name)/learning_time).round(2)})".center(6)}
   respond(output)
 end
 
 total = DRSkill.list.select { |item| item.current - item.baseline > 0 }.map { |skill| DRSkill.gained_exp(skill.name) }.inject(&:+).round(2)
 
 respond("   TOTAL: #{total}")
-respond("   Learning For: #{((Time.now - DRSkill.start_time) / 60.0 / 60).round(1)} hours")
+respond("   Learning For: #{learning_time} hours")
```

Comment:
I'd leave the round(1) call here, so that we retain the greater precision for determining ranks above.